src/client.js

```diff
@@ -166,6 +168,15 @@ class CommandoClient extends discord.Client {
 		return undefined;
 	}
 
+	get mentionPrefix() {
+		if(!this.options.mentionPrefix) return null;
```

Comment:
Why does it return `null` and not `false`?

Reply:
Set return type to default to true.
Check commit e4fd992eb18c8aff112024622de99879d444f401

Reply:
Why? Now it's always true, regardless of what I set it to.

Reply:
See f59bd78
Removed the check at all, because I check for null in the constructor and default it to true.
Also forget, it was a boolean value, there ! will not work the same as with other datatypes, with just checking if it exists or not.